collect_app/src/test/java/org/odk/collect/android/utilities/MediaUtilsTest.kt

```diff
@@ -0,0 +1,80 @@
+package org.odk.collect.android.utilities
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.widget.Toast
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+import org.odk.collect.android.BuildConfig
+import org.odk.collect.androidshared.system.IntentLauncher
+import org.robolectric.shadows.ShadowToast
+import java.io.File
+
+@RunWith(AndroidJUnit4::class)
+class MediaUtilsTest {
+
+    private val context = ApplicationProvider.getApplicationContext<Context>()
+    private lateinit var intentLauncher: IntentLauncher
```

Comment:
These can all be initialized up here and avoid the `lateinit`.

Reply:
Done